src/components/proofs/Proof.tsx

```diff
@@ -27,10 +27,15 @@ import classnames, {
   maxWidth,
   space,
   width,
+  wordBreak,
 } from 'classnames/tailwind'
 import getEtherscanAddressUrl from 'helpers/getEtherscanAddressUrl'
 
-const proofName = classnames(display('flex'), flex('flex-1'))
+const proofName = classnames(
+  display('flex'),
+  flex('flex-1'),
+  wordBreak('break-words')
```

Comment:
You may use `hyphens` prop for the `ContractName` component. But this is an option too

Reply:
Let's keep it like this for now